frontend/src/routes/AdminRoutes.js

```diff
@@ -6,6 +6,7 @@ import { useBoundStore } from "../stores/useBoundStore"
 import AdminNav from "../components/Admin/AdminNav/AdminNav"
 import AdminTitleBar from "../components/Admin/AdminTitleBar/AdminTitleBar"
 import UsersPage from "../pages/UsersPage/UsersPage"
+import UpdateUser from "../pages/CreateUpdateUser/UpdateUser"
```

Comment:
Remember to update this import when you change the name of the CreateUpdateUser folder